test/test_distributions.py

```diff
@@ -425,13 +425,14 @@ def get_sp_dist(jax_dist):
         np.array([0.0, 1.0]),
         np.array([[1.0], [2.0]]),
     ),
-    T(dist.TruncatedNormal, -1.0, 0.0, 1.0),
-    T(dist.TruncatedNormal, 1.0, -1.0, np.array([1.0, 2.0])),
+    T(_TruncatedNormal, 0.0, 1.0, -1.0, None),
+    T(_TruncatedNormal, -1.0, np.array([1.0, 2.0]), 1.0, None),
```

Comment:
nit: I think you can use `dist.TruncatedNormal` and remove `_TruncatedNormal`.

Reply:
I'd tried this originally, but it makes the tests a lot more complicated because `dist.TruncatedNormal` takes keyword-only arguments, and so lots of tests needed adjusting to have both `params` and `kw_params` in the signature. IMO it's much simpler like this, but I'll change it if you prefer

Reply:
You are right. Thanks for explaining!